images/base/Dockerfile

```diff
@@ -22,7 +22,12 @@
 # https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/
 
 ARG BASE_IMAGE="ubuntu:18.04"
+
+FROM golang:latest as fetch_ctr
+RUN go get github.com/containerd/containerd/cmd/ctr
```

Comment:
so @Random-Liu says that docker 18.09+ should actually have ctr already? it's part of the containerd release